rootfs/usr/local/bin/launch.sh

```diff
@@ -73,8 +73,7 @@ fi
 
 if [ "$PHP82" = "true" ]; then PHP_INI_SCAN_DIR=/data/php/82/conf.d php-fpm82 -c /data/php/82 -y /data/php/82/php-fpm.conf -FOR; fi &
 if [ "$PHP83" = "true" ]; then PHP_INI_SCAN_DIR=/data/php/83/conf.d php-fpm83 -c /data/php/83 -y /data/php/83/php-fpm.conf -FOR; fi &
-if [ "$LOGROTATE" = "true" ] && [ "$GOA" = "false" ]; then sleep 1m; while true; do logrotate --verbose --state /data/etc/logrotate.status /etc/logrotate; sleep 25h; done; fi &
-if [ "$LOGROTATE" = "true" ] && [ "$GOA" = "true" ]; then sleep 1m; while true; do killall goaccess; sleep 10s; logrotate --verbose --state /data/etc/logrotate.status /etc/logrotate; sleep 25h; done; fi &
+if [ "$LOGROTATE" = "true" ]; then while true; do sleep $LOGROTATEDELAY; logrotate --verbose --state /data/etc/logrotate.status /etc/logrotate; done; fi &
```

Comment:
what is the advantage of LOGROTATEDELAY instead of 1m?

Reply:
It lets you run logrotate at a shorter interval and waits a reasonable time before its 1st run. This way by default waits 25 hrs before running logrotate for the 1st time and then 25 hrs for each subsequent run. If you are working on a dev environment you can lower it to 3 hrs and it'll run the logrotate sequence every 3 hrs but only action once every day. A live environment 25hrs for the 1st run and every subsequent one is fine but shorter is required if you are restarting all the time.

Reply:
but why should it run every 3 hours? logroate only rotates after 24 hours

Reply:
Yes 24hr after the last run BUT if you restart the server after 21 hrs for example and leave it running the logs will be 46 hrs old before they rotate with your settings. If you restart at 23hrs the logs will be 48hrs old.
Using 3hrs rebooting at 23hrs (worst case) the logs rotate at 26hrs old.

Reply:
but we could also leave it as it is, which means that the log is rotated one minuted after start and then every 25h

Reply:
Which means goaccess could be killed just as its started. This is what we were trying to avoid.

Reply:
but 1m should be enough?